boefjes/boefjes/config.py

```diff
@@ -2,43 +2,58 @@
 from pathlib import Path
 from typing import Optional
 
-from pydantic import AnyHttpUrl, BaseSettings, PostgresDsn
+from pydantic import AmqpDsn, AnyHttpUrl, BaseSettings, Field, IPvAnyAddress, PostgresDsn
 
 
 class RuntimeConfiguration(Enum):
     LOCAL = "local"
 
 
 class Settings(BaseSettings):
-    base_dir: Path = Path(__file__).parent.resolve()
-    log_cfg: Path = Path(__file__).parent / "logging.json"
+    base_dir: Path = Field(Path(__file__).parent.resolve())
+    log_cfg: Path = Field(Path(__file__).parent / "logging.json", description="Path to the logging configuration file")
```

Comment:
This will list the build directory of the documentation as default value which isn't really nice. There are also other places where the default value isn't even correct, such as when it is `set()` or with enums. Is there a way to override the default value in the documentation?

Reply:
I've changed it such that the BASE_DIR is fixed when compiling documentation. As for the other variables: I don't think there's a good way to do this except for either modifying the jinja template or manually specifying an if-statement alternative for DOCS=1. We could do either, but given that it's not critical I'd suggest moving this to future work.